### PR TITLE
Fixed the Outline Only option in the montage dialogs

### PR DESCRIPTION
**Pull request by tplatt77**
Since the combo box defaults to Outline Only, the index changed function is never triggered. To fix it, I set the display type to the 0 index item (which is Outline Only at the moment).

### CODE REVIEW

**Review by joeykleingers — CHANGES_REQUESTED** [review]
This still needs some work to get the "Outline Only" feature to function how we intend.  Outline Only should not register or stitch the montage tiles, it should display the tiles exactly how they're displayed in Side-By-Side except using outlines instead of displaying the actual tile data.

**Comment by tplatt77** [comment]
Additional changes were made in the IMFViewer repo to complete requested changes.

File: SIMPLVtkLib/Dialogs/ImportDREAM3DMontageDialog.cpp [hunk 1]
```diff
@@ -111,6 +111,8 @@ void ImportDREAM3DMontageDialog::setupGui()
   m_Ui->spacingY->setValidator(new QDoubleValidator);
   m_Ui->spacingZ->setValidator(new QDoubleValidator);
 
+  setDisplayType(static_cast<AbstractImportMontageDialog::DisplayType>(0));
```
Comment (joeykleingers) [inline SIMPLVtkLib/Dialogs/ImportDREAM3DMontageDialog.cpp]: This should be `setDisplayType(AbstractImportMontageDialog::DisplayType::Outline);`

File: SIMPLVtkLib/Dialogs/ImportFijiMontageDialog.cpp [hunk 2]
```diff
@@ -98,6 +98,8 @@ void ImportFijiMontageDialog::setupGui()
   m_Ui->spacingY->setValidator(new QDoubleValidator);
   m_Ui->spacingZ->setValidator(new QDoubleValidator);
 
+  setDisplayType(static_cast<AbstractImportMontageDialog::DisplayType>(0));
```
Comment (joeykleingers) [inline SIMPLVtkLib/Dialogs/ImportFijiMontageDialog.cpp]: See above

File: SIMPLVtkLib/Dialogs/ImportGenericMontageDialog.cpp [hunk 3]
```diff
@@ -96,6 +96,8 @@ void ImportGenericMontageDialog::setupGui()
   m_Ui->spacingY->setValidator(new QDoubleValidator);
   m_Ui->spacingZ->setValidator(new QDoubleValidator);
 
+  setDisplayType(static_cast<AbstractImportMontageDialog::DisplayType>(0));
```
Comment (joeykleingers) [inline SIMPLVtkLib/Dialogs/ImportGenericMontageDialog.cpp]: See above

**Review by joeykleingers — CHANGES_REQUESTED** [review]
The only fix I see is changing all the setDisplayType calls to use the actual enumeration value.  Other than that, it looks good!

**Review by joeykleingers — APPROVED** [review]

**Pull request merged**